src/OrchardCore.Modules/OrchardCore.Redis/Startup.cs

```diff
@@ -47,6 +47,12 @@ public override void ConfigureServices(IServiceCollection services)
                 configurationOptions.CertificateValidation += IgnoreCertificateErrors;
             }
 
+            var allowAdmin = section.GetValue<bool?>("AllowAdmin", null);
+            if (allowAdmin.HasValue)
+            {
+                configurationOptions.AllowAdmin = allowAdmin.Value;
+            }
```

Comment:
This change is unrelated, but it is nice to have when using Aspire to configure Redis. I've been using something like this in an Aspire app host:

```csharp
var redis = builder.AddRedis("Redis");

builder.AddProject<Projects.OrchardCore_Cms_Web>("orchardcore-cms-web")
    .WithHttpsEndpoint(5001)
    .WithEnvironment("OrchardCore__OrchardCore_Redis__Configuration", redis)
    .WithEnvironment("OrchardCore__OrchardCore_Redis__AllowAdmin", "true")
    .WaitFor(redis);
```